### ISSUE TITLE
Disable Certain Options

### ISSUE THREAD

**Issue by Viper714** [acemod/ACE3#6741]
**Arma 3 Version:** `1.88` (stable / rc / dev)
**CBA Version:** `3.9.0` (stable / dev + commit hash)
**ACE3 Version:** `3.12.5` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
This is more of a question than an issue. I was not able to find a proper place to add. IF it belongs elsewhere please let me know. This is the first time that we've initiated the ACE 3 MOD on our dedicated server. We love the ACE 3 environment, but wanted to see if there was a way to disable certain option and go back to the vanilla version. Would like our members and new player to ease into the realism and not have all of them from the start. Here are some of the options that i would like to revert back to vanilla:

**Gun Scopes** 
Can you go back to the vanilla version where you adjust the azimuth with standard "page Up" and "Page Down" keys and it changes in the upper right corner of your HUD? 

**Vehicle Repair, Rearm and Refueling**
Would like the ability to just pull up to a a repair, refuel or ammo vehicle like in the vanilla version to accomplish the task. Is it possible to change that?

**What Happened to standard NATO Vehicles??**
With the activation of the ACE 3 MOD, noticed that vehicles were renamed. What happened to the  Slammer, Cheetah, and Bobcat. There are other vehicles as well that have name changes as well. Do I have something loaded incorrectly?

We are currently using the KP Liberation CTI maps which we enjoy and love the realism of ACE 3, but as I mentioned, would prefer to ease into the realism components. 

Thank you ion advance!!!

**Comment by commy2** [comment]
Delete the corresponding component files from your repository. Make sure everyone is using the same repository.

They are inside the @ace3/addons folder and conveniently named ace_scopes, ace_repair, ace_rearm, ace_refuel(ing?) and ace_realisticnames.pbo

If you're using Steam, it will download the files again, because Steam is garbage.

**Comment by PabstMirror** [comment]
ace scopes has a settings for "simplifiedZeroing" that should do what you want
check out https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System for info on how to use
rearm/repair and refuel all require config changes so the pbos need to be removed to restore vanilla
realistic names just renames stuff, everything should still be there

**Comment by Viper714** [comment]
Thank you for the information. I've tried to remove the pbo's in the addon folder, but like said, steam keeps redownloading. I changed the scope settings to simplified Zeroing,  and it works great. Trying to understand the CBA settings systems, but is a lot to take in for this noob. If there is a step by step on how to make changes in the rearm / repair / refuel, I would really appreciate it. It would be great if they would be available in the ingame server addon settings...
